atlassian-prod/etc/description.mustache

```diff
@@ -1,22 +1,20 @@
-The snowdrop team is really pleased to contact you as we will release in 12 weeks a new Snowdrop BOM based on the following Spring Boot version: *{{release.version}}*
+The Snowdrop team is really pleased to inform you that we're planning to release a new Snowdrop BOM on {{release.date}} based on Spring Boot *{{release.version}}*.
 
-{{#isComponent}}
-As product owner of the following {{type}}: *{{component.name}}*, we would like to know if you plan to release a new {{type}} for the version: *{{component.version}}*
-that we will start to test the: *{{release.dueDateFormatted}}*.
-{{/isComponent}}
-{{#isStarter}}
-As product owner of the following {{type}}: *{{component.name}}*, we would like to know if you plan to release a new {{type}}
-that we will start to test the: *{{release.dueDateFormatted}}*.
-{{/isStarter}}
```

Comment:
The message to pass to a team/project producing a starter is different for a component. Why, for most of the starters, we dont know the version they will produce for the new starter like also the component packaged within the starter. e.g. resteasy starter packaging resteasy component

This is why we use as word either component vs starter and if there is a version, then it will be indicated within the description. Example: Hibernate update version as released within the Spring Boot BOM

@metacosm

Reply:
For me, this aspect doesn't matter since I'd rather list **all** the artifacts we want the product team to validate for a given Spring Boot version. Also, as I've mentioned in the JIRA ticket, the version part of the text is confusing: what version is it? Is it the version we want to use? The version we currently use? The version Spring Boot uses? If we want to keep the version there, we need to make it explicit.

Reply:
> For me, this aspect doesn't matter since I'd rather list **all** the artifacts we want the product team to validate for a given Spring Boot version.

Such information could then be added within the cloned JIRA issue created for a release instead of being added to the JIRA issue created under JWS, KEYCLOAK, ... projects. Make sense ? @metacosm 
Remark : I will check if we can easily append text to a JIRA description

Reply:
@cmoulliard no, it doesn't make sense… 😅 I even more confused now about what you mean than previously… 😭